Formula/jose.rb

```diff
@@ -0,0 +1,31 @@
+class Jose < Formula
+  desc "C-language implementation of Javascript Object Signing and Encryption"
+  homepage "https://github.com/latchset/jose"
+  url "https://github.com/latchset/jose/releases/download/v8/jose-8.tar.bz2"
+  sha256 "24e3d71e3da5a7913ab3c299381d76dfde488d91cb108b1a9527454bf1e9dc51"
+
+  head do
+    url "https://github.com/latchset/jose.git"
+
+    depends_on "libtool" => :build
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "jansson"
+  depends_on "lzlib"
+  depends_on "openssl"
+
+  def install
+    system "autoreconf", "-fiv" if build.head?
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system bin/"jose", "alg"
```

Comment:
```
bash-3.2$ cat ./tests/test-suite.log
==================================
   jose 8: tests/test-suite.log
==================================

# TOTAL: 23
# PASS:  20
# SKIP:  0
# XFAIL: 0
# FAIL:  3
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: jose-jwk-exc
==================

++ mktemp -d
usage: mktemp [-d] [-q] [-t prefix] [-u] template ...
       mktemp [-d] [-q] [-u] -t prefix 
+ tmpdir=
FAIL jose-jwk-exc (exit status: 1)

FAIL: jose-jws-sig
==================

++ jose alg -k sign
+ ALGS='ES256
ES384
ES512
HS256
HS384
HS512
PS256
PS384
PS512
RS256
RS384
RS512'
++ mktemp -d
usage: mktemp [-d] [-q] [-t prefix] [-u] template ...
       mktemp [-d] [-q] [-u] -t prefix 
+ tmpdir=
FAIL jose-jws-sig (exit status: 1)

FAIL: jose-jwe-enc
==================

++ jose alg -k wrap
+ WRAP='A128GCMKW
A128KW
A192GCMKW
A192KW
A256GCMKW
A256KW
dir
ECDH-ES
ECDH-ES+A128KW
ECDH-ES+A192KW
ECDH-ES+A256KW
PBES2-HS256+A128KW
PBES2-HS384+A192KW
PBES2-HS512+A256KW
RSA-OAEP
RSA-OAEP-256
RSA1_5'
++ jose alg -k encr
+ ENCR='A128CBC-HS256
A128GCM
A192CBC-HS384
A192GCM
A256CBC-HS512
A256GCM'
++ mktemp -d
usage: mktemp [-d] [-q] [-t prefix] [-u] template ...
       mktemp [-d] [-q] [-u] -t prefix 
+ tmpdir=
FAIL jose-jwe-enc (exit status: 1)

bash-3.2$
```
So it's the same as
https://unix.stackexchange.com/questions/30091/fix-or-alternative-for-mktemp-in-os-x
https://github.com/facebook/flow/pull/3833
https://github.com/elasticdog/transcrypt/issues/34